README.md

```diff
@@ -468,6 +522,13 @@ for await (const file of $.fs.expandGlob("**/*.ts")) {
 }
 ```
 
+Re-export of @sindresorhus's strip-ansi:
```

Comment:
There is already a strip ansi in this module so we don't need to include another dependency (search for `strip_ansi`). We can just export that functionality. You'll need to update to the latest main in order to get it synchronous instead of async due to https://github.com/dsherret/dax/pull/72.